### ISSUE TITLE
Narrator is reading items in NavigationView's PaneFooter incorrectly.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#2108]
**Describe the bug**

Narrator is reading the items in NavigationView menu items incorrectly. Specifically items that are in the PaneFooter. Can reproduce in the XamlControlsGallery app.

This is a known issue with NavigationView where the footer and settings items have incorrect automation properties.

**Steps to reproduce the bug**

1.  Start with the code for Xaml-Controls-Gallery https://github.com/microsoft/Xaml-Controls-Gallery/tree/master/
2.       Open the project in Visual Studio 2019.
3.       Edit the file XamlControlsGallery\ControlPages\NavigationViewPage.xaml
4. See attached image for ref.
  At line 100 add: IsSettingsVisible="false"
  At line 109 add a new footer item:
```xaml
<muxc:NavigationView.PaneFooter>
    <muxc:NavigationViewItem Content="Footer Item" Tag="SamplePage5" Icon="Play" />
</muxc:NavigationView.PaneFooter>
```
Open Windows Narrator (run Narrator, or open via Windows Settings->Ease Of Access)

5.       Build and Run your Xaml-Controls-Gallery project
6.       In Xaml-Controls-Gallery select Navigation, then NavigationView, click the “Download” in the inner “Sample Page 1” screen. Note Narrator reads it as “Menu Item 4, 4 of 4” (correctly).
7.       Press down arrow on your keyboard to highlight out new footer button. Narrator reads it as “Footer Item, 5 of 5” (incorrect). It should read it as “Footer Item, 1 of 1”.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/76572808-51d52e00-6478-11ea-8f03-fae04f3d75ff.png)

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/25465741)

**Comment by YuliKl** [comment]
Feels like this problem may go away when #375 is implemented.

**Comment by chingucoding** [comment]
The behavior is still present, however we now have NavigationView FooterMenuItems, so putting items inside the PaneFooter is not our preferred or recommended way anymore. @YuliKl What are your thoughts on this issue given that putting items in the FooterMenuItems results in the desired narrator behavior?

**Comment by Felix-Dev** [comment]
> The behavior is still present, however we now have NavigationView FooterMenuItems, so putting items inside the PaneFooter is not our preferred or recommended way anymore. 

I don't think that's correct. There are different use cases for putting items in the PaneFooter or the FooterMenuItems section. Some items might require selections, others don't. Those who don't will still be put inside the PaneFooter and as such should be read correctly by the Narrator. (FooterMenuItems section is for menu items which need to participate in the selection).

Groove is an example app showing that the new FooterMenuItems API in no way "retires" the PaneFooter API.
